### PR TITLE
[JN-1007] updating consent form population

### PR DESCRIPTION
**Pull request by devonbush**
#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)
This switches the load attempt order for consent forms -- first it will attempt to load it as a survey now.  It also updates the demo population so that consents are populated as surveys.

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

1.  repopulate demo
2. go to sandbox.demo.localhost:3001
3. sign in as newbie@test.com, password  `d2p4eva!`
4. fill out the consent form
5. confirm you do not see errors in the logs related to loading consent form

### CODE REVIEW

File: ui-participant/src/hub/consent/ConsentView.tsx [hunk 1]
```diff
@@ -140,20 +140,20 @@ export default function ConsentView() {
...
   const loadForm = async () => {
     try {
-      const response = await Api.fetchConsentAndResponses({
-        studyShortcode,
-        enrolleeShortcode: enrollee.shortcode, stableId, version
+      await Api.fetchSurveyAndResponse({
+        studyShortcode, enrolleeShortcode: enrollee.shortcode, stableId, version, taskId
       })
-      setFormAndResponses(response)
+      // it's a survey -- view it there
```
Comment (connorlbark) [inline ui-participant/src/hub/consent/ConsentView.tsx:146]: nit - this comment seems more confusing now up top, maybe `first, try loading as a survey`

File: populate/src/test/java/bio/terra/pearl/populate/extract/PortalExtractTest.java [hunk 2]
```diff
@@ -51,9 +51,9 @@ public void testExtractDemoPortal() throws Exception {
         assertThat(portalLanguageService.findByPortalEnvId(sandboxPortalEnv.getId()), hasSize(3));
 
         // confirm all templates got repopulated
-        assertThat(surveyService.findByPortalId(restoredPortal.getId()), hasSize(12));
+        assertThat(surveyService.findByPortalId(restoredPortal.getId()), hasSize(13));
         assertThat(studyService.findByPortalId(restoredPortal.getId()), hasSize(1));
-        assertThat(consentFormService.findByPortalId(restoredPortal.getId()), hasSize(1));
+        assertThat(consentFormService.findByPortalId(restoredPortal.getId()), hasSize(0));
```
Comment (connorlbark) [inline populate/src/test/java/bio/terra/pearl/populate/extract/PortalExtractTest.java:56]: woo!

**Review by connorlbark — APPROVED** [review]

File: ui-participant/src/hub/consent/ConsentView.tsx (hunk 1 above)
Comment (devonbush) [reply ui-participant/src/hub/consent/ConsentView.tsx:146]: concur -- but I'll leave it as-is for now since we're going to nuke all this code soon anyway

**Review by ssettipalli — APPROVED** [review]
looks good to me.

**Pull request merged**